packages/host/app/components/operator-mode/container.gts

```diff
@@ -406,22 +411,52 @@ export default class OperatorModeContainer extends Component<Signature> {
   // to have its own background URL. Also need to consider how to treat adjoining
   // stacks that have the same background image (consider 4 stacks, where 2
   // adjacent stacks have the same background image)
-  fetchBackgroundImageURL = trackedFunction(this, async () => {
-    let bottomMostCard = this.stacks[0]?.[0];
-    let realmInfo;
-    if (bottomMostCard) {
-      if (bottomMostCard.type !== 'card') {
-        throw new Error(
-          `bug: the bottom most card for a stack cannot be a contained card`
-        );
-      }
-      realmInfo = await this.cardService.getRealmInfo(bottomMostCard.card);
-    }
-    return realmInfo?.backgroundURL;
+  fetchBackgroundImageURLs = trackedFunction(this, async () => {
```

Comment:
It looks like the comment above could be updated now since this does handle different background URLs 🎉